packages/rxjs-traces/src/devtools.ts

```diff
@@ -55,24 +56,51 @@ window.addEventListener('message', (event: MessageEvent) => {
     data.source === 'rxjs-traces-devtools' &&
     data.type === 'receive'
   ) {
-    if (extensionSubscription) {
-      extensionSubscription.unsubscribe();
-    }
-    extensionSubscription = eventHistory$.subscribe(({ type, payload }) => {
-      window.postMessage(
-        {
-          source: 'rxjs-traces',
-          type,
-          payload: prepareForTransmit(payload),
-        },
-        window.location.origin
-      );
-    });
+    window.postMessage(
+      {
+        source: 'rxjs-traces',
+        type: 'event-history',
+        payload: prepareForTransmit(pastHistory),
+      },
+      window.location.origin
+    );
   }
 });
 
+declare class WeakRef<T extends object> {
+  constructor(target?: T);
+  deref(): T | undefined;
+}
```

Comment:
Magic ✨ 

A bit of explaination for this: In the current version, as we could only have one instance of devtools, what rxjs-traces did was to create backpressure by keeping the event history until someone subscribes, through the `publishReplayOnce` operator. When someone subscribed, that history was cleaned of and back pressure was removed.

Now we don't have this assumption: It might happen that someone has some devtools installed in their browser and opens a project that also connects some standalone devtools.

To solve this, I thought on keeping all the history throughout the lifetime of the app, to replay it back to new subscribers later on.

The problem is that, although the events themselves are smallish (yep, in time it will keep increasing in size, but rxjs-traces shouldn't be used in prod), the content of the events can be huge, and if our internal history is keeping a reference to those objects, they won't get GC.

So the solution is to use WeakRef in the history: This is an object that holds a weak reference to an object, that you can `deref` at anytime. If that object was GCed, you'll get undefined. This way, we can hold these references while allowing the engine to get rid of potentially huge objects, and if an instance of devtools subscribes late, it will show that on that time there was a new value, but it was GCed.

Note that this is only available in Firefox, Chrome and Edge. Maybe we need an alternative for other browsers (Safari?).